### PR TITLE
Tests: Make container count to be configurable

### PR DESCRIPTION
**Pull request by mtuchkova**

### CODE REVIEW

File: src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy [hunk 1]
```diff
@@ -8,7 +8,7 @@ import org.prebid.server.functional.util.ObjectMapperWrapper
 class PbsServiceFactory {
 
     private static final Map<Map<String, String>, PrebidServerContainer> containers = [:]
-    private static final int MAX_CONTAINERS_COUNT = 5
+    private static final Integer MAX_CONTAINERS_COUNT = Integer.valueOf(System.getProperty("max.containers.count"))
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy:11]: `int` please. You will still get an exception if you pass in null.

**Review by Net-burst — APPROVED** [review]
LGTM with minor remark.

File: src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy [hunk 2]
```diff
@@ -4,11 +4,13 @@ import org.prebid.server.functional.service.PrebidServerService
 import org.prebid.server.functional.testcontainers.container.NetworkServiceContainer
 import org.prebid.server.functional.testcontainers.container.PrebidServerContainer
 import org.prebid.server.functional.util.ObjectMapperWrapper
+import org.prebid.server.functional.util.PBSUtils
 
 class PbsServiceFactory {
 
     private static final Map<Map<String, String>, PrebidServerContainer> containers = [:]
-    private static final int MAX_CONTAINERS_COUNT = 5
+    private static final Integer MAX_CONTAINERS_COUNT = Integer.valueOf(
```
Comment (snahornyi) [inline src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy:12]: Can we use parseInt(String num)?

**Review by snahornyi — CHANGES_REQUESTED** [review]

**Review by Net-burst — APPROVED** [review]
LGTM

**Review by snahornyi — APPROVED** [review]

**Pull request merged**